extensions/amp-analytics/0.1/visibility-impl.js

```diff
@@ -161,7 +161,7 @@ export function getElement(selector, el, selectionMethod) {
   } else if (selectionMethod == 'scope') {
     return el.parentElement.querySelector(selector);
   } else if (selector[0] == '#') {
-    return el.ownerDocument.getElementById(selector.slice(1));
+    return ampdoc.getElementById(selector.slice(1));
```

Comment:
Is that for the `closest` polyfill or for both? Do we just pass the ampdoc to the call and stop closest ancestor traversal when we reach it?